docs/community/governance.md

```diff
@@ -0,0 +1,106 @@
+
+**All** of KOSS meetings have to be conducted in the knowledge of an Executive Head. Executive Heads have to take steps to prevent any abuse of power or harassment.
+
+Executive Heads regularly consult with the Advisors. They must do it before introducing any change with high impact.
+
+They approve Pull Requests to https://github.com/kossiitkgp/accounts.
+
+### 1.2 Executive Members
+
+Executive Members do not take the lead but have experience in working with KOSS and the erstwhile Executive Heads. Their primary objectives are:
+
+- Involving Core Team members through projects/hackathons to improve their skills
+- Supporting Executive Heads when necessary
+
+They act as resources of KOSS. They offer their suggestions constantly throughout the year.
+
+They do not make any decision for KOSS without consulting the Executive Heads, or unless told to do so.
+
+### 2. Core Team members
+
+> Members in their late first and early second year
+
+Core Team members always represent the next generation of KOSS and FOSS culture at IIT Kharagpur. They actively volunteer to learn and conduct events, workshops, classes, etc. CTMs also learn how to work in a team. “Community is more important than Code,” and the core team should take this seriously.
+
+At the end of the session, there is a week-long discussion with the Core Team members where they put their ideas forward on topics such as ‘Events,’ ‘Community Standards,’ ‘New hires,’ ‘Meetings and Communications’, etc. After these discussions, the former Executive Heads select the new Executive Heads and Executive Members and terminate the membership of people who are not a cultural fit to work with the Executives.
+
+### 3. Advisors
+
+> Members who have been an Executive before and are still on campus. Late third year onwards.
+
+Advisors, in general, have more technical experience. They actively suggest better technologies to be used in KOSS. They also make sure the Executive Heads are not over-prioritizing anything and are relaxed. They continuously take updates from the Executive Heads about what is going in KOSS. They intervene when asked by an Executive to resolve any conflict. They can also lead meetings if Executive Heads do not object.
+
+One of the Advisors acts as the Treasurer of KOSS and oversees the finances. See [Accounts](/docs/community/accounts).
+
+### 4. Alumni
+
+“Members who have graduated from KGP”
+
+The role has no formal duties, as of now.
+
+
+## Guidelines
+
+### Guidelines for Executive Heads -
+
+1. Do not miss a meeting unless extremely required (health or out of station). Your lack of involvement will psychologically disolve your responsibilities. Schedule meetings according to the availability of all the Executive Heads.
+1. Be in sync among yourselves. Physically meet at-least once a week. Slack and Emails do not take importance over your coordination. You will build an amazing friendship with this, which will remain beyond KOSS.
```

Comment:
I have doubts regarding this "Physically meet at-least once", I don't want anything in this document that people will ignore. This seems like that.

Reply:
:+1: On this from my side.

Reply:
I agree on this with @rava-dosa. We should tell them the benefit and also a responsibility of being in sync among each other but not in a forceful way like "Physically meet at least once a week"(although the intention was not "forceful" here but the new batch will take this in that way and probably ignore it as something "too forced to do").

Reply:
Changed it to "Frequently meet with each other."